staging/src/k8s.io/client-go/kubernetes/typed/core/v1/fake/fake_pod_expansion.go

```diff
@@ -58,11 +58,12 @@ func (c *FakePods) GetLogs(name string, opts *v1.PodLogOptions) *restclient.Requ
 }
 
 func (c *FakePods) Evict(eviction *policy.Eviction) error {
-	action := core.CreateActionImpl{}
-	action.Verb = "create"
+	action := core.GenericActionImpl{}
```

Comment:
/lgtm cancel

[https://github.com/kubernetes/kubernetes/blob/master/pkg/client/clientset_generated/internalclientset/typed/core/internalversion/fake/fake_pod_expansion.go](https://github.com/kubernetes/kubernetes/blob/master/pkg/client/clientset_generated/internalclientset/typed/core/internalversion/fake/fake_pod_expansion.go)

oh, fix this for the internal pod client too please 🙂

Reply:
It seems the internal PodExpansion doesn't have an Evict method

Reply:
yeah.. but it's not intended. fine to keep a minimal change

Reply:
So do you want me to add the Evict for that one or not? Since it’s not there right now